src/shared/components/VideoPlayer/VideoPlayer.style.tsx

```diff
@@ -46,6 +46,13 @@ export const Container = styled.div<ContainerProps>`
     height: ${sizes(16)} !important;
     align-items: center;
 
+    @media screen and (max-width: ${breakpoints.small}) {
```

Comment:
I may have failed to communicate that, but we do the mobile-first media queries everywhere in the app. So we first define the default values for the smallest screen (without media query) and then provide overrides via media queries with `min-width`